app/controllers/admin/suppliers_controller.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+class Admin::SuppliersController < Admin::BaseController
+  skip_after_action :verify_authorized, only: :index
+  skip_after_action :verify_policy_scoped, except: :index
+
+  def index
+    lead_providers = policy_scope(LeadProvider)
+    delivery_partners = policy_scope(DeliveryPartner)
+    sorted_suppliers = (lead_providers + delivery_partners).sort_by(&:name)
+    @suppliers = Kaminari.paginate_array(sorted_suppliers).page(params[:page]).per(20)
+    @page = @suppliers.current_page
+    @total_pages = @suppliers.total_pages
```

Comment:
How strictly do we feel about 'not accessing any methods / fields on objects in our views'? Not necessarily something to do on this PR, but probably worth agreeing on it, perhaps making sure we all know it, and following it religiously later.

Reply:
If the methods are public, well tested and free of side effects I can't see why calling them in the views would be a bad idea. For me calculations, objects instantiations, long chains of method calls etc, in the views are the red flag